### ISSUE TITLE
[0.08] Recipe/Quests broken due to Content Tweaker ID changes

### ISSUE THREAD

**Issue by Hjaldar** [Vyraal1/Interaction#24]
It looks like the update from 0.07 has shifted around some metadata values for Content Tweaker items. My Reinforced Stone Ingots turned into Fiery Infused Rods. I can fix that myself no problems, this is just for information purposes.

The quest Reinforced Stone in Sky - Main is also affected. It is showing the wrong icon (fiery infused rod) and the required item is also wrong. Both are simple fixes.

Alge

**Comment by Hjaldar** [comment]
Redstone Gear, which has no current recipe but is used in recipes for XU2 scanner and redstone clock, changed to Fiery Infused Dust.

**Comment by Vyraal1** [comment]
The quest one will need to wait on the fixing, or if someone wants to PR it, mostly because I usually leave quests at a low priority (DJJ wanted to get started on them) until everything at a tier is reviewed.
